database/seeds/BodySeeder.php

```diff
@@ -48,36 +29,324 @@ public function run()
+            'city'          =>  'A Coruna',
+        ]);
+
     	Body::create([
-            'type_id'       =>  2,
+            'type_id'       =>  1,
             'address_id'    =>  1,
-    		'name'			=>	'AEGEE-Europe',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'AEU' //No clue if this is actually correct.
+    	    'name'	    =>	'AEGEE-A Coruna',
+            'email'         =>  'aegee.coruna@gmail.com',
+	    'phone'	    =>  '',
+            'legacy_key'    =>  'TMP' //TODO
     	]);
+	
+        Address::create([
+            'country_id'    =>  81,
+            'street'        =>  'Templergraben 55',
+            'zipcode'       =>  '52056',
+            'city'          =>  'Aachen',
+        ]);
 
     	Body::create([
             'type_id'       =>  1,
             'address_id'    =>  2,
-    		'name'			=>	'AEGEE-Enschede',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'ENS' //No clue if this is actually correct.
+    	    'name'	    =>	'AEGEE-Aachen',
+            'email'         =>  'info@aegee-aachen.org',
+	    'phone'	    =>  '+49 2418097121',
+            'legacy_key'    =>  'TMP' //TODO
     	]);
 
+	Address::create([ //TODO: Address unknown
+            'country_id'    =>  220,
+            'street'        =>  '',
+            'zipcode'       =>  '',
+            'city'          =>  'Adana',
+        ]);
+
     	Body::create([
             'type_id'       =>  1,
             'address_id'    =>  3,
-    		'name'			=>	'AEGEE-Dresden',
-            'email'         =>  'info@aegee-dresden.org',
-            'legacy_key'    =>  'DRE' //No clue if this is actually correct.
+    	    'name'	    =>	'AEGEE-Adana',
+            'email'         =>  'adanaaegee@gmail.com',
+	    'phone'	    =>  '+90 5071378919',
+            'legacy_key'    =>  'TMP' //TODO
     	]);
 
+	Address::create([ //TODO: Address unknown
+            'country_id'    =>  106,
+            'street'        =>  '',
+            'zipcode'       =>  '',
+            'city'          =>  'Agrigento',
+        ]);
+
     	Body::create([
-            'type_id'       =>  3,
-            'address_id'    =>  1,
-    		'name'			=>	'Comité Directeur',
-            'email'         =>  'headoffice@aegee.org',
-            'legacy_key'    =>  'DVD' //No clue if this is actually correct.
+            'type_id'       =>  1,
+            'address_id'    =>  4,
+    	    'name'	    =>	'AEGEE-Agrigento',
+            'email'         =>  'aegee92100@gmail.com',
+	    'phone'	    =>  '',
+            'legacy_key'    =>  'TMP' //TODO
+    	]);
+
+	Address::create([
+            'country_id'    =>  202,
+            'street'        =>  'Carretera de San Vicente',
+            'zipcode'       =>  '03690',
+            'city'          =>  'Alicante',
+        ]);
+
+    	Body::create([
+            'type_id'       =>  1,
+            'address_id'    =>  5,
+    	    'name'	    =>	'AEGEE-Alicante',
```

Comment:
Should not be though, are you indenting using spaces or tabs? We previously agreed upon using indentation of 4 spaces throughout the core, my guess is that setting your editor to use this would fix it.